applications/FluidDynamicsApplication/python_scripts/trilinos_navier_stokes_embedded_solver.py

```diff
@@ -87,29 +78,24 @@ def GetDefaultSettings(cls):
     def __init__(self, model, custom_settings):
         self._validate_settings_in_baseclass=True # To be removed eventually
         # Note: deliberately calling the constructor of the base python solver (the parent of my parent)
```

Comment:
I think with little refactoring this hack is no longer needed-

Reply:
Agree.